values-with-pvc.yaml

```diff
@@ -0,0 +1,173 @@
+global:
+  # registry secret for private registry from where example project needs to be pulled
+  registryPullSecrets:
+    - "quay-registry-secret"
+
+runtime-bundle:
+  nameOverride: rb
+  extraEnv: |
+    - name: SERVER_PORT
+      value: "8080"
+    - name: SERVER_SERVLET_CONTEXTPATH
+      value: "{{ tpl .Values.ingress.path . }}"
+    - name: SERVER_USEFORWARDHEADERS
+      value: "true"
+    - name: SERVER_TOMCAT_INTERNALPROXIES
+      value: ".*"
+    - name: MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE
+      value: "*"
+    - name: "ACTIVITI_CLOUD_APPLICATION_NAME"
+      value: "{{ .Release.Name }}"
+    - name: ACT_KEYCLOAK_RESOURCE
+      value: "{{ .Release.Name }}"
+    - name: KEYCLOAK_USERESOURCEROLEMAPPINGS
+      value: "true"
+    - name: SPRING_ACTIVITI_PROCESSDEFINITIONLOCATIONPREFIX
+      value: file:/root/{{ .Release.Name }}/1/processes/"
+    - name: PROJECT_MANIFEST_FILE_PATH
+      value: "file:/root/{{ .Release.Name }}/1/simpleapp.json"
+    - name: SPRING_ACTIVITI_PROCESSDEFINITIONLOCATIONPREFIX
+      value: "file:/root/{{ .Release.Name }}/1/processes/"
```

Comment:
if you parameterise the volume mount version "1" also here should reflect that.

I would see very well this version at global level on the top of the values yaml

Reply:
yes this is correct I will update this